source/ShipInfoDisplay.cpp

```diff
@@ -45,10 +45,11 @@ ShipInfoDisplay::ShipInfoDisplay(const Ship &ship, const PlayerInfo &player, boo
 
 
 // Call this every time the ship changes.
-void ShipInfoDisplay::Update(const Ship &ship, const PlayerInfo &player, bool descriptionCollapsed)
+// scrollingPanel is for panels such as the shipyard and outfitter that cannot overflow because they scroll.
```

Comment:
```suggestion
// Panels that have scrolling abilities are not limited by space, allowing more detailled attributes.
```
Thats better I think